changelogs/fragments/7263-proxmox-return-vmid-and-taskid.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+    - proxmox - add vmid and taskid to return codes
```

Thanks for your contribution! Here's a first comment:
```suggestion
    - proxmox - add ``vmid`` and ``taskid`` to return values (https://github.com/ansible-collections/community.general/pull/7263).
```